cluster/gce/gci/configure.sh

```diff
@@ -126,9 +126,9 @@ function validate-hash {
   local -r file="$1"
   local -r expected="$2"
 
-  actual=$(sha1sum ${file} | awk '{ print $1 }') || true
+  actual=$(sha512sum ${file} | awk '{ print $1 }') || true
```

Comment:
should this check if the passed in hash matches the sha1 or sha512sum of the file?

Reply:
no, this script is bundled with the releases and the callers of this function will not pass sha1 in the future.